README.md

```diff
@@ -150,6 +151,7 @@ Example commands:
   git clone git@github.com:myaccount/my-project.git
 ```
 --->
+> git clone https://github.com/waelelsafty07/vet.git
```

Comment:
- [ ] In the README file please do the following:
1. Fix the issues page
2. Remove the Live Demo link
3. Fill out each section of the README file with the corresponding data of this specific project
4. Add your correct personal social links
This is for a professional README file and repo. 🙏